x-pack/platform/plugins/shared/ml/server/models/annotation_service/annotation.ts

```diff
@@ -9,7 +9,7 @@ import Boom from '@hapi/boom';
 import { each, get } from 'lodash';
 import type { IScopedClusterClient } from '@kbn/core/server';
 
-import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import type { estypes } from '@elastic/elasticsearch';
```

Comment:
LOC 286 in this file also contains deprecated `body`

Reply:
Good point! Addressed in [`7282412` (#208776)](https://github.com/elastic/kibana/pull/208776/commits/72824126ff23367a75bf055a0398fce7a4aa0058)